tests/p2p_test_peers.py

```diff
@@ -0,0 +1,16 @@
+import subprocess
+
+class P2PTestPeers:
+    Debug = False
+
+    hosts = ["localhost", "127.0.0.1"]
```

Comment:
Can we expect remote host support in a future pull request?  `eosio-launcher` is currently being reworked to be compatible with Dawn 3.